docs/entity_registry_index.md

```diff
@@ -20,3 +20,11 @@ Good sources for a unique ID:
 - latitude/longitude
 
 If a device has a single serial but provides multiple entities, combine the serial with unique identifiers for the entities. For example, if a device measures both temperature and humidity, you can uniquely identify the entities using `{serial}-{sensor_type}`.
+
+```python
+# Example, inside a component
```

Comment:
This is not necessary if we link to the docs for entity properties above

Reply:
Sure. I see this section is more of an overview, but when I found and read it, I could not tell what to do. Will you add the link to entity properties, @balloob or how do we proceed?

Reply:
I put that in my suggestion in the other comment?